CMakeLists.txt

```diff
@@ -737,6 +737,7 @@ set(HDR_PUBLIC
     include/event2/event.h
     include/event2/event_compat.h
     include/event2/event_struct.h
+    include/event2/event_watcher.h
```

Comment:
I think there are non-CMake builds I need to support as well... can somebody help direct me to the other build files I need to update? Thanks!

Reply:
autotools (which is primary build system for libevent right now) -- `Makefile.am`

*P.S. I do not ask you to fix manually written files for nmake, since I'm going to remove them*